### ISSUE TITLE
Outdated Documentation on Official Website

### ISSUE THREAD

**Issue by gohrner** [cryptomator/cryptomator.github.io#102]
If you click "Documentation" in the website page footer, you end up at https://docs.cryptomator.org/en/latest/ , which still contains the documentation of Cryptomator 1.7, while 1.10 is the current version.

**Comment by infeo** [comment]
Actually, the documentation is "up to date" (meaning that there is no newer version). 

The 1.7 is displayed due to the organizing the documentation versions by branches. We chose this scheme longer time ago. Since Cryptomator 1.7 we switched the app versioning model allowing more feature releases. Unfortunately, this would mean creating a lot of duplicate branches (because there aren't a lot of changes in between), hence, we decided to just update the 1.7 branch for now.
